site/src/components/sections/SectionInfo.vue

```diff
@@ -2,39 +2,255 @@
+                :key="college"
+                class="college"
+              >
+              {{ college }}
+              </li>
+            </ul>
+          </template>
+          <template v-if="prerequisiteData.restrictions.college.may_not_be">
+            <div class="font-weight-bold">
+              Not allowed for the following schools:
+            </div>
+            <ul>
+              <li
+                v-for="college in prerequisiteData.restrictions.college.may_not_be"
+                :key="college"
+                class="college"
+              >
+              {{ college }}
+              </li>
+            </ul>
+          </template>
+        </template>
+        <template v-if="prerequisiteData.restrictions.major">
+          <template v-if="prerequisiteData.restrictions.major.must_be">
+            <div class="font-weight-bold">
+              Restricted to the following majors:
+            </div>
+            <ul>
+              <li
+                v-for="major in prerequisiteData.restrictions.major.must_be"
+                :key="major"
+                class="major"
+              >
+              {{ major }}
+              </li>
+            </ul>
+          </template>
+          <template v-if="prerequisiteData.restrictions.major.may_not_be">
+            <div class="font-weight-bold">
+              Not allowed for the following majors:
+            </div>
+            <ul>
+              <li
+                v-for="major in prerequisiteData.restrictions.major.may_not_be"
+                :key="major"
+                class="major"
+              >
+              {{ major }}
+              </li>
+            </ul>
+          </template>
+        </template>
+        <template v-if="prerequisiteData.restrictions.field_of_study">
+          <template v-if="prerequisiteData.restrictions.field_of_study.must_be">
+            <div class="font-weight-bold">
+              Restricted to the following fields of study (major or minor):
+            </div>
+            <ul>
+              <li
+                v-for="field_of_study in prerequisiteData.restrictions.field_of_study.must_be"
+                :key="field_of_study"
+                class="field_of_study"
+              >
+              {{ field_of_study }}
+              </li>
+            </ul>
+          </template>
+          <template v-if="prerequisiteData.restrictions.field_of_study.may_not_be">
+            <div class="font-weight-bold">
+              Not allowed for the following fields of study (major or minor):
+            </div>
+            <ul>
+              <li
+                v-for="field_of_study in prerequisiteData.restrictions.field_of_study.may_not_be"
+                :key="field_of_study"
+                class="field_of_study"
+              >
+              {{ field_of_study }}
+              </li>
+            </ul>
+          </template>
+        </template>
       </template>
-      <br />
-      <br />
```

Comment:
Why was this removed?

Reply:
Yes, the <ul> tags I used add the necessary spacing themselves. Without removing the <br /> tags, it created a very long gap in the section info card. I moved these from here to after the prerequisite display, prior to the conditional lists, in the latest commits.